src/libmongoc/src/mongoc/mongoc-uri.c

```diff
@@ -893,7 +895,8 @@ mongoc_uri_split_option (mongoc_uri_t *uri,
    mongoc_lowercase (key, lkey);
 
    /* Initial DNS Seedlist Discovery Spec: "A Client MUST only support the
-    * authSource and replicaSet options through a TXT record, and MUST raise an
+    * authSource, replicaSet, and loadBalanced options through a TXT record, and
+    * MUST raise an
```

Comment:
Text wrapping is premature if you want to consolidate

Reply:
Done.